js/blocks/controls/post.js

```diff
@@ -1,63 +1,15 @@
-import FetchInput from './fetch-input';
```

Comment:
This shouldn't change the Post control, only refactor most of it into `content.js`.